lms/static/js/student_account/tos_modal.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Modal for displaying the Terms of Service in the Register page.
+ */
+
+(function($, gettext) {
+    'use strict';
+
+    var buildModal = function(link, modalClass, contentClass, closeButtonClass) {
+        // Create the modal container with a title, close button and content area
+        var closeLabel = gettext('Close'),
+            titleId = 'modal-header-text',
+            modal = $('<div>', {
+                'class': modalClass
+            }),
+            content = $('<div>', {
+                'class': contentClass,
+                'role': 'dialog',
+                'aria-modal': true,
+                'aria-labelledby': titleId
+            }),
+            closeButton = $('<button>', {
+                'aria-label': closeLabel,
+                'class': closeButtonClass
+            }),
+            title = $('<h2>', {
+                'id': titleId
+            });
+        closeButton.text(closeLabel);
+        content.append(closeButton);
+        title.text($(link).text());
+        content.append(title);
+        modal.append(content);
+        return modal;
+    };
+
+    var buildIframe = function(link, contentSelector) {
+        // Create an iframe with contents from the link and set its height to match the content area
+        return $('<iframe>', {
+            src: link.href,
+            load: function() {
+                var iframeHead = $(this).contents().find('head');
+                // Overwrite styles in child page hiding top navigation and footer
```

Comment:
@replaceafill Nit: This sounds like the styles in the child page are hiding the top navigation and footer, and that that's the behavior we're trying to override. How about changing the wording as follows:

> Overwrite styles in child page to hide top navigation and footer